examples/ssh/linux_services.js

```diff
@@ -0,0 +1,133 @@
+/** Name: Linux Services Monitoring
+ * Description: Monitors the status of all the services on a Linux machine
+ *   
+ * Communication protocol is SSH, using the Linux bash command.
+ * 
+ * Tested on Linux with the following specs:
+ *      - Debian Linux 10
+ *      - bash version 5.0.3(1)
+ * 
+ * Requires:
+ *      - systemctl
+ *      - sed, grep, awk, and tail
+ *      - PLEASE NOTE: it requires to be rut as root
```

Comment:
Typo, its not rut its "run". Also its not the root user but a user from the sudoers group